centos/centos-7.x/centos-7.6-hpc/install_mellanoxofed.sh

```diff
@@ -18,5 +18,10 @@ KERNEL=${KERNEL[-1]}
 yum install -y kernel-devel-${KERNEL}
 ./${MOFED_FOLDER}/mlnxofedinstall --kernel $KERNEL --kernel-sources /usr/src/kernels/${KERNEL} --add-kernel-support --skip-repo --skip-unsupported-devices-check --without-fw-update
 
+# Issue: Module mlx5_ib belong to a kernel which is not a part of MLNX
+# Resolution: set FORCE=1 in /etc/init.d/openibd 
+# This causes openibd to ignore the kernel difference but relies on weak-updates
+sed -i '147 s/FORCE=0/FORCE=1/' /etc/init.d/openibd
+
 # Restarting openibd
 /etc/init.d/openibd restart
```

Comment:
Instead of this, should we just add FORCE=1 while doing openibd restart?

Otherwise, we will be forcing it every time we do reload.

Reply:
Resolved